samples/unity-of-bugs/Assets/Scripts/SentryNativeBridge.m

```diff
@@ -0,0 +1,121 @@
+#import <Sentry/Sentry.h>
+
+void SentryNativeBridgeAddBreadcrumb(const char* timestamp, const char* message, const char* type, const char* category, int* level) {
+    NSLog(@"Native Bridge: Adding breadcrumb.");
+    
+    if (timestamp == NULL && message == NULL && type == NULL && category == NULL && level == NULL) {
+        NSLog(@"Native Bridge: Breadcrumb empty. Dropping it.");
+        return;
+    }
+    
+    [SentrySDK configureScope:^(SentryScope * _Nonnull scope) {
+        SentryBreadcrumb *breadcrumb = [[SentryBreadcrumb alloc] init];
+        
+        if (timestamp != NULL) {
+            NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+            [dateFormatter setDateFormat:NSCalendarIdentifierISO8601];
+            breadcrumb.timestamp = [dateFormatter dateFromString:[NSString stringWithCString:timestamp encoding:NSUTF8StringEncoding]];
+        }
+
+        if (message != NULL) {
+            breadcrumb.message = [NSString stringWithCString:message encoding:NSUTF8StringEncoding];
+        }
+
+        if (type != NULL) {
+            breadcrumb.type = [NSString stringWithCString:type encoding:NSUTF8StringEncoding];
+        }
+        
+        if (category != NULL) {
+            breadcrumb.category = [NSString stringWithCString:category encoding:NSUTF8StringEncoding];
+        }
+        
+        if (level != NULL) {
```

Comment:
`m`: As level is an `int` it should never be `NULL`. Maybe it's worth checking if the passed `int` can be mapped to `SentryLevel`.